### PR TITLE
Info mortal / immortal extrinsics

### LINKED ISSUE #5066: Add warning about re-using reaped accounts
https://wiki.polkadot.network/docs/learn-accounts#existential-deposit-and-reaping

A re-activated reaped account is vulnerable to replay attacks, since it resets the nonce. We should add a warning to this section.

### PR DESCRIPTION
**Pull request by filippoweb3**

### CODE REVIEW

File: docs/learn/learn-extrinsics.md [hunk 1]
```diff
@@ -48,6 +48,25 @@ Here are some key differences between the different types of extrinsics:
 - Inherents exist to address the need of adding some data to a block, whereas signed or unsigned
   transactions exist to potentially change the state of the blockchain.
 
+### Mortal and Immortal Extrinsics
+
+Extrinsics can be mortal (i.e. valid within a defined block interval) or immortal (i.e. always
+valid). By default and for security reasons, all extrinsics submitted using the
+[Polkadot-JS UI](../general/polkadotjs-ui.md) and
+[browser extension](../general/polkadotjs.md#polkadot-js-extension) will be mortal. This will avoid
```
Comment (laboon) [inline docs/learn/learn-extrinsics.md]: Does the extension actually matter here? It's just a signer, doesn't the Polkadot-JS App actually generate the extrinsic and the extension just signs it?

I just don't want users to think that anything signed with the browser extension is definitely mortal.

File: docs/learn/learn-extrinsics.md [hunk 2]
```diff
@@ -48,6 +48,25 @@ Here are some key differences between the different types of extrinsics:
...
+the chance of being a victim of replay attacks after
```
Comment (laboon) [inline docs/learn/learn-extrinsics.md]:
```suggestion
the chance of being a victim of a replay attack after
```

File: docs/learn/learn-extrinsics.md [hunk 3]
```diff
@@ -48,6 +48,25 @@ Here are some key differences between the different types of extrinsics:
...
+[reaping an account](./learn-accounts.md#existential-deposit-and-reaping).
+
+Reaping an account resets the nonce value. If all signed transactions until the nonce before the
```
Comment (laboon) [inline docs/learn/learn-extrinsics.md]: Add "Replay attacks work as follows:" or similar to the beginning of this paragraph.

File: docs/learn/learn-extrinsics.md [hunk 4]
```diff
@@ -48,6 +48,25 @@ Here are some key differences between the different types of extrinsics:
...
+reaping event were immortal, once the account is refunded, all transactions can be replayed because
+the information about the signatures for those transactions already exists on-chain, and the
```
Comment (laboon) [inline docs/learn/learn-extrinsics.md]:
```suggestion
a valid transaction for that account and nonce already exists and has been signed. 
```

File: docs/learn/learn-extrinsics.md [hunk 5]
```diff
@@ -48,6 +48,25 @@ Here are some key differences between the different types of extrinsics:
...
+refunded account will look new to the present chain state. There is no need for the attacker to know
```
Comment (laboon) [inline docs/learn/learn-extrinsics.md]:
```suggestion
There is no need for the attacker to know
```

File: docs/learn/learn-extrinsics.md [hunk 6]
```diff
@@ -48,6 +48,25 @@ Here are some key differences between the different types of extrinsics:
...
+your private key.
+
+Making a transaction mortal will (almost) ensure that replay attacks are not valid, with the only
```
Comment (laboon) [inline docs/learn/learn-extrinsics.md]:
```suggestion
Making a transaction mortal will almost certainly ensure that replay attacks are not possible, with the only
```

File: docs/learn/learn-extrinsics.md [hunk 7]
```diff
@@ -48,6 +48,25 @@ Here are some key differences between the different types of extrinsics:
...
+particular case is if the account gets reaped immediately after submitting a mortal transaction, and
```
Comment (laboon) [inline docs/learn/learn-extrinsics.md]:
```suggestion
exception being if the account is reaped and then re-funded shortly after submitting a mortal transaction, and
```

File: docs/learn/learn-extrinsics.md [hunk 8]
```diff
@@ -48,6 +48,25 @@ Here are some key differences between the different types of extrinsics:
...
+within the mortal window (i.e. the block interval), an attacker replays that transaction.
```
Comment (laboon) [inline docs/learn/learn-extrinsics.md]:
```suggestion
then an attacker replays that transaction within the mortality window (i.e., the specified the block interval).
```

**Review by laboon — CHANGES_REQUESTED** [review]
Just some minor edits for clarification/wording

File: docs/learn/learn-extrinsics.md (hunk 1 above)
Comment (filippoweb3) [reply docs/learn/learn-extrinsics.md]: Right, I will remove those tools and keep a general tone

**Comment by laboon** [comment]
lgtm

**Pull request merged**